tests/gateway/test_client.py

```diff
@@ -471,3 +471,22 @@ def test_cllient_query_with_disallowed_param(mixed_gateway):
 
     with pytest.raises(HTTPError, match=".*The parameter 'model' is not permitted.*"):
         gateway_client.query(route=route.name, data=data)
+
+
+@mock.patch(
+    "mlflow.gateway.constants.MLFLOW_GATEWAY_CLIENT_QUERY_TIMEOUT_SECONDS", new_callable=lambda: 1
+)
```

Comment:
same comment on this line: https://github.com/mlflow/mlflow/pull/9069/files#r1263128942